### ISSUE TITLE
[Bug]: iOS isn't tracked

### ISSUE THREAD

**Issue by hajiboy95** [oddbit/flutter_facebook_app_events#364]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

At the  Meta for Developer [Ads-Helptool webpage ](https://developers.facebook.com/tools/app-ads-helper/) It always tells me that last iOS installation is not available. Android installation seems to be tracked however. 

I tried the emulator as well as uploading it and test it with beta tester for my app. boeth doesn't get tracked

### Expected Behavior

app install should be tracked 

### Steps To Reproduce

Add this to your info.pfile 
`<key>CFBundleURLTypes</key>
	<array>
		<dict>
			<key>CFBundleTypeRole</key>
			<string>Editor</string>
			<key>CFBundleURLSchemes</key>
			<array>
				<string>$(CFBundleURLSchemes)</string>
			</array>
		</dict>
		<dict>
			<key>CFBundleURLSchemes</key>
			<array>
				<string>fb$(FB_APP_ID)</string>
			</array>
		</dict>
	</array>
	<key>FacebookAppID</key>
	<string>$(FB_APP_ID)</string>
	<key>FacebookClientToken</key>
	<string>$(FACEBOOK_CLIENT_TOKEN)</string>
	<key>FacebookDisplayName</key>
	<string>$(FB_DISPLAY_NAME)</string>`
	
	I also added google login, that's why I've got a few more lines in the CFBUNDLEURLSCHEME section 

### Platform

iOS

### Flutter Doctor

```shell
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 3.22.2, on macOS 14.5 23F79 darwin-arm64, locale de-DE)
[✓] Android toolchain - develop for Android devices (Android SDK version 34.0.0)
[✓] Xcode - develop for iOS and macOS (Xcode 15.4)
[✓] Chrome - develop for the web
[✓] Android Studio (version 2023.3)
[✓] VS Code (version 1.90.1)
[✓] Connected device (4 available)            
[✓] Network resources

• No issues found!
```

**Comment by hajiboy95** [comment]
I didn't import the package since I only want to track app installs for advertising later on. 
Furthermore, my app hasn't been approved yet by the meta suite since it's not published yet so I don't have a proper link I could forward (however I've got an bundle ID which I inserted. Maybe that's the reason? Does the app need to be published officially first in the case of iOS? For google it seems to be working before official release
